lib/fastlane/plugin/firebase_app_distribution/client/firebase_app_distribution_api_client.rb

```diff
@@ -96,16 +107,37 @@ def get_app(app_id, app_view = 'BASIC')
         App.new(response.body)
       end
 
+      # Get AAB info (Android apps only)
+      #
+      # args
+      #   project_number - Firebase project number
+      #   app_id - Firebase App ID
+      #
+      # Throws a user_error if the app hasn't been onboarded to App Distribution
+      def get_aab_info(project_number, app_id)
+        begin
+          aab_name = aab_info_name(project_number, app_id)
```

Comment:
This looks like it might be double duty...isn't `aab_info_url(project_number, app_id)` == `aab_name`? - why do you need to pass in a name param here if that's what the URL already is?

Reply:
Nope, they are different (one is prefixed with `/v1/`.